en/docs/install-and-setup/setup/user-stores/setting-up-a-userstore-in-icp.md

```diff
@@ -0,0 +1,306 @@
+| `user.password` | Password                                    | Yes                      |
+| `user.is_admin` | Whether the user is given admin privileges  | Default value is `false` |
+
+!!! Tip
+    Set `user.is_admin` to `true` to grant admin privileges to a user in the file-based user store.
+
+## Disable the file-based user store
+
+To disable the file-based user store, add the following to the `<ICP_HOME>/conf/deployment.toml` file:
+
+```toml
+[internal_apis.file_user_store]
+enable = false
+```
+
+!!! note
+    See the [Integration Control Plane Configuration Catalog]({{base_path}}/reference/config-catalog-integration-control-plane/#single-sign-on) documentation to connect to the dashboard via Single Sign-On (SSO).
+
+## Configure an LDAP user store
+
+!!! info "Before you begin"
+    - See the documentation of your LDAP provider for instructions on setting up the LDAP.
+    - Disable the file-based user store as shown below using the `<ICP_HOME>/conf/deployment.toml` file.
+
+         ```toml
+         [internal_apis.file_user_store]
+         enable = false
+         ```
+
+Follow the steps below to connect the ICP to your LDAP user store.
+
+1. Open the `deployment.toml` file stored in the `<ICP_HOME>/conf/` directory.
+2. Add the following configurations and update the required values.
+
+### Read write LDAP
+
+```toml
+[user_store]
+connection_url = "ldap://localhost:10389"
+connection_name = "uid=admin,ou=system"
+connection_password = "admin"
+user_search_base = "ou=Users,dc=wso2,dc=org"
+type = "read_write_ldap"
+```
+
+### Read only LDAP
+ 
+```toml
+[user_store]
+connection_url = "ldap://localhost:10389"
+connection_name = "uid=admin,ou=system"
+connection_password = "admin"
+user_search_base = "ou=Users,dc=wso2,dc=org"
+type = "read_only_ldap"
+read_groups = "true"
+```
+
+The parameters used above are explained below.
+
+<table>
+	<tr>
+		<th>Parameter</th>
+		<th>Value</th>
+	</tr>
+	<tr>
+		<td>
+			<code>connection_url</code>
+		</td>
+		<td>
+			The URL for connecting to the LDAP. If you are connecting over LDAPS (secured LDAP), you need to import the certificate of the user store to the truststore (<code>client-truststore.jks</code> by default). See the instructions on how to <a href="{{base_path}}/install-and-setup/setup/security/importing-ssl-certificate/#importing-ssl-certificates-to-a-truststore">add certificates to the truststore</a>.
+		</td>
+	</tr>
+	<tr>
+		<td>
+			<code>connection_name</code>
+		</td>
+		<td>
+			The username used to connect to the user store and perform various operations. This user needs to be an administrator in the user store. That is, the user requires write permission to manage add, modify users and perform search operations on the user store. The value you specify is used as the DN (Distinguish Name) attribute of the user who has sufficient permissions to perform operations on users and roles in LDAP.
```

Comment:
Incorrect wording? @DinithiDiaz 

`The username used to` => `The username is used to`

`That is, the user requires write permission to manage add, modify users and perform search operations on the user store. ` => `That means the user requires write permission to manage, add, modify users, and perform search operations on the user store.`